[assert_compare.assertion.1] line 15 assertion Func.all(5): FAILURE
VERIFICATION FAILED
